### PR TITLE
Fix Image.getSize() according to image orientation

### PR DESCRIPTION
**Pull request by doochik**
This is fix for issue https://github.com/facebook/react-native/issues/14097

## Test Plan
I've prepared images with different exif orientation (https://yadi.sk/d/is_xcEEo3MRAxF)
0-4 should have size 200x100
5-8 should have size 100x200

### CODE REVIEW

File: Libraries/Image/RCTImageLoader.m [hunk 1]
```diff
@@ -716,10 +716,30 @@ - (RCTImageLoaderCancellationBlock)getImageSizeForURLRequest:(NSURLRequest *)ima
         CGSize size;
         if ([imageOrData isKindOfClass:[NSData class]]) {
             NSDictionary *meta = RCTGetImageMetadata(imageOrData);
-            size = (CGSize){
-                [meta[(id)kCGImagePropertyPixelWidth] doubleValue],
-                [meta[(id)kCGImagePropertyPixelHeight] doubleValue],
-            };
+
+            double imageOrientation = [meta[(id)kCGImagePropertyOrientation] doubleValue];
```
Comment (shergin) [inline Libraries/Image/RCTImageLoader.m]: We have to use `integerValue` method, `NSInteger` type and no conversion.

Comment (doochik) [reply Libraries/Image/RCTImageLoader.m]: oops, my bad. Fix it!

**Comment by doochik** [comment]
@shergin could you cherry-pick this to 0.48 ?

**Comment by shergin** [comment]
@doochik Everyone can! You should advocate this in special issue somewhere here. 😄

**Pull request closed without merging**